### PR TITLE
Canary

### PR DESCRIPTION
**Pull request by antweiss**

### CODE REVIEW

**Review by itaysk — COMMENTED** [review]
I thought we said the application itself (the python server will be in a different repo, no?
Also, please add some content to the readme.md. It's going to be shown in the examples catalog. you can copy some summary from the post and link to it.

**Comment by itaysk** [comment]
Also, please add some content to the readme.md. It's going to be shown in the examples catalog. you can copy some summary from the post and link to it.

File: canary-release/cfexample.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+title: canary-release
```
Comment (itaysk) [inline canary-release/cfexample.yaml]: Title should be user readable, spaces are ok.

File: canary-release/cfexample.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+description: canary release pattern implementation with Codefresh pipelines and K8S
+sourceUrl: http://github.com/Codefresh-Examples/Examples/canary-release
+baseUrl: http://github.com/Codefresh-Examples/Examples/canary-release
```
Comment (itaysk) [inline canary-release/cfexample.yaml]: wrong baseurl.. for example http://github.com/Codefresh-Examples/Examples/canary-release/README.md is not a valid url. use github raw url here

File: canary-release/cfexample.yaml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+labels:
+  - canary
+  - delpoyment
+  - kubernetes
+icon: "icon.png"
+body: README.md
+supportingFiles:
```
Comment (itaysk) [inline canary-release/cfexample.yaml]: sorry we recently changed supporting files to 'extraFiles' please update.

**Review by itaysk — CHANGES_REQUESTED** [review]
comments inline and in thread

File: canary-release/cfexample.yaml (hunk 2 above)
Comment (antweiss) [reply canary-release/cfexample.yaml]: I can fix this after the PR is merged. Right now there is no actual link to the file in the original repo.

**Comment by antweiss** [comment]
This has no python files. Only docker files, yamls and test execution scripts.

File: canary-release/cfexample.yaml [hunk 4]
```diff
@@ -0,0 +1,17 @@
+title: Canary Release with Kubernetes
+description: canary release pattern implementation with Codefresh pipelines and K8S
+sourceUrl: https://github.com/Codefresh-Examples/Examples/canary-release
+baseUrl: https://github.com/Codefresh-Examples/Examples/tree/master/canary-release
```
Comment (itaysk) [inline canary-release/cfexample.yaml]: this also won't work. just use: `https://raw.githubusercontent.com/Codefresh-Examples/Examples/master/canary-release`

**Pull request merged**